.gitignore

```diff
@@ -66,3 +66,5 @@ venv.bak/
 
 # Conda-forge recipe generated by grayskull
 pyospackage/*
+# setuptools scm generates this file - it should never be committed to history
+src/pyospackage/_version.py
```

Comment:
This file shouldn't be committed to history - it is generated on the fly when the package is built.